packages/venia-ui/lib/components/MiniCart2/minicart.css

```diff
@@ -0,0 +1,80 @@
+.root {
+    /* The root aside element takes up the whole screen. */
+    position: fixed;
+    left: 0;
+    top: 0;
+    height: 100%;
+    width: 100%;
+
+    /* It is hidden by default. */
+    opacity: 0;
+    visibility: hidden;
+
+    /* It animates to being closed. */
+    transition-duration: 192ms;
+    transition-timing-function: var(--venia-anim-out);
```

Comment:
The timing feels off for this interaction. I would expect the background to blur as the mini cart appears but instead it seems like the cart appears and then the background is blurred. This is probably because this CSS doesn't get applied until the component is rendered.

[![Image from Gyazo](https://i.gyazo.com/92a6d43409bf30a9c1fc316d5cc6f621.gif)](https://gyazo.com/92a6d43409bf30a9c1fc316d5cc6f621)

Reply:
Yeah, these animations should be synced. Also, the dropdown should probably translate a little bit. I'll link you a sandbox, but something like a `-8px` to `0px` translation would work.

Reply:
This was actually resolved when I took the `blur` out. Perhaps it only calculates that after animating the foreground? Anyway, the component is always rendered (it's part of the `CartTrigger` now), and we just toggle its `display` property.